core/jazz_metrics/components/utils.js

```diff
@@ -200,7 +200,7 @@ function getAssetsObj(assetsArray, userStatistics) {
     } else {
       // type not found
       newAssetArr.push({
-        "message": `Asset type not found `,
+        "message": `Asset type not found: ${assetType}`,
```

Comment:
Isn't `assetType` null now? Why log it explicitly as null again?

Reply:
No, `assetType` is required. We get metrics for all assets of a given assetType. Asset Name is not passed to the API.

Reply:
Hmm, do you need this `else` block  then?

Reply:
Removed else block and changed the last else if to else.